website/react-magma-docs/src/pages/api/stepper.mdx

```diff
@@ -0,0 +1,592 @@
+          borderRadius: '6px',
+          margin: '20px 0 0',
+          padding: '20px',
+        }}
+      >
+        {currentStep === 0 && <div>Step Content One</div>}
+        {currentStep === 1 && <div>Step Content Two</div>}
+        {currentStep === 2 && <div>Step Content Three</div>}
+        {currentStep === 3 && <div>Step Content Four</div>}
+        {currentStep === 4 && <div>Steps Completed</div>}
+      </Container>
+
+      <Container style={{ padding: '20px 0' }}>
+        <Button style={{ marginRight: '4px' }} onClick={handleOnPrevious}>
+          Previous
+        </Button>
+        <Button onClick={currentStep >= 4 ? handleFinish : handleOnNext}>
+          {currentStep >= 4 ? 'Finish' : 'Next'}
+        </Button>
+      </Container>
+    </>
+  );
+}
+```
+
+## Custom Step label
+
+When using `summaryView`, the option to change the count title may be done with the `stepDescriptionLabel` which takes a string and replaces the default "Step" label.
+
+```typescript
+import React from 'react';
+import { Container, Stepper, Step } from 'react-magma-dom';
+
+export function Example() {
+  const [currentStep, setCurrentStep] = React.useState(0);
+
+  const handleOnNext = () => {
+    if (currentStep !== 4) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+  const handleOnPrevious = () => {
+    if (currentStep !== 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+  const handleFinish = () => {
+    if (currentStep === 4) {
+      setCurrentStep(0);
+    }
+  };
+
+  return (
+    <>
+      <Stepper
+        summaryView
+        currentStep={currentStep}
+        stepDescriptionLabel="Module"
+      >
+        <Step label="First Step" secondaryLabel="Description One" />
+        <Step label="Second Step" secondaryLabel="Description Two" />
+        <Step label="Third Step" secondaryLabel="Description Three" />
+        <Step label="Fourth Step" secondaryLabel="Description Four" />
+      </Stepper>
+
+      <Container
+        style={{
+          background: '#F5F5F5',
+          borderRadius: '6px',
+          margin: '20px 0 0',
+          padding: '20px',
+        }}
+      >
+        {currentStep === 0 && <div>Step Content One</div>}
+        {currentStep === 1 && <div>Step Content Two</div>}
+        {currentStep === 2 && <div>Step Content Three</div>}
+        {currentStep === 3 && <div>Step Content Four</div>}
+        {currentStep === 4 && <div>Steps Completed</div>}
+      </Container>
+
+      <Container style={{ padding: '20px 0' }}>
+        <Button style={{ marginRight: '4px' }} onClick={handleOnPrevious}>
+          Previous
+        </Button>
+        <Button onClick={currentStep >= 4 ? handleFinish : handleOnNext}>
+          {currentStep >= 4 ? 'Finish' : 'Next'}
+        </Button>
+      </Container>
+    </>
+  );
+}
+```
+
+## Completed Step Description
+
+When using `summaryView`, after all of the steps are complete, the `completedStepDescription` prop takes a string and changes the title.
```

Comment:
I noticed in this example that Fourth Step is still visible. Should that disappear when all steps are completed?
![image](https://github.com/cengage/react-magma/assets/91160746/39d56be1-8314-4e3f-a5d8-483b3a9be9a0)